python/tank/bootstrap/manager.py

```diff
@@ -1006,10 +1006,6 @@ def _start_engine(self, tk, engine_name, entity, progress_callback=None):
         :returns: Started :class:`~sgtk.platform.Engine` instance.
         """
 
-        # Log a user activity metric saying which engine and entity are bootstrapping
```

Comment:
why are we not logging bootstrap metrics anymore? apologies in advance if this is explained further town in the PR :) 

We definitely want to track which users are using the bootstrap API. this is super interesting.

Reply:
Would be good for @robblau to clarify what he wants to do with these existing metrics that weren't called out in the spreadsheet. I was not expecting we would remove any existing metrics.

Reply:
ah! i see. I think understanding bootstrap use (and plugin use) is probably useful!

Reply:
I removed based on my understanding from a cgat with Rob.

"Rob Blau"
aug. 31st 
"We don’t really have any coverage in the spreadsheet for the app store or general change management.  Having that in amplitude can definitely be saved for later, so it is fine just leaving this metric out of the new system."

"Rob Blau 
[sept 1st 2017, 10:41 AM] 
Hey all. :thumbs_up: to getting rid of unspec'ed events, but could I get a list of them in case it was an oversight in the ticket and we should actually log it?